src/api/cpp/cvc5.h

```diff
@@ -4372,6 +4486,40 @@ class CVC5_EXPORT Solver
    */
   Statistics getStatistics() const;
 
+  /* .................................................................... */
+  /* Optimization                                                         */
+  /* .................................................................... */
+
+  /**
+   * Run optimization for the objectives
+   * @param objCombination for multi-objective optimization,
+   *  indicate whether it's a BOX optimization,
+   *  a LEXICOGRAPHIC optimization or a PARETO optimization
+   * @return a pair <Result, vector<OptimizationResult>>,
+   *  where Result indicates the overall optimization result,
+   *  for BOX, if any of the objective is UNSAT or UNKNOWN,
+   *    it's UNSAT/UNKNOWN,
+   *  for LEXICOGRAPHIC, if the assertions are UNSAT/UNKNOWN,
+   *    it's UNSAT/UNKNOWN,
+   *  for PARETO, if the assertions are UNSAT
+   *  or the possible solutions are exhausted, it's UNSAT,
```

Comment:
> or the possible solutions are exhausted, it's UNSAT

This makes it sound like the user should be calling this method multiple times to get the next solution. I don't think that this is what we want to suggest.

Reply:
forgot to update...